update-help-messages.sh

```diff
@@ -10,10 +10,18 @@ INCLUDE_DIR="docs/include"
 
 # Given a single commands, print the help text, and wrap the output in rst style
 # code block format
-function print_help() {
-    local cmd="$1"
-    local help_text
+function print_help_to_file() {
+    local cmd="$*"
+
+    # Replace spaces in the file name with dashes
+    local file_path
+    if [ "$cmd" == "" ]; then
+        file_path="$INCLUDE_DIR/pgcopydb.rst"
+    else
+        file_path="$INCLUDE_DIR/${cmd// /-}.rst"
+    fi
 
+    local help_text
     # shellcheck disable=SC2086
     help_text="$(pgcopydb $cmd --help 2>&1)"
```

Comment:
Can you double-quote `"${cmd}"` and remove the shellcheck disable comment? Also please use `${name}` notation, makes it easier to review/maintain.